SEP-0008.md

```diff
@@ -0,0 +1,151 @@
+    Examples: 0.9.0a1, 1.2.3rc1, 2.0.0b10
+    
+1.  Development release numbers can be used to specify versions taken from
+    version control or not publicly released. These versions MUST NOT be
+    uploaded to indexes such as PyPI or anaconda.org. A development release MAY
+    be denoted using .devM where M is a non-negative integer number. Development
+    releases are ordered by their numerical component, immediately before the
+    corresponding release (and before any pre-releases with the same release
+    segment), and following any previous release (including any post-releases).
+    Development releases MAY also be used in conjunction with pre-release
+    numbering.
+    Examples: 1.0.0.dev25, 1.2.3a1.dev4532 1.0.0rc2.dev3456
+
+1.  Precedence refers to how versions are compared to each other when
+    ordered. Precedence MUST be calculated by separating the version
+    into major, minor, patch and pre-release identifiers in that order.
+    Precedence is
+    determined by the first difference when comparing each of these
+    identifiers from left to right as follows: Major, minor, and patch
+    versions are always compared numerically. Example: 1.0.0 < 2.0.0
+    < 2.1.0 < 2.1.1. When major, minor, and patch are equal, a
+    pre-release version has lower precedence than a normal version. Example:
+    1.0.0a1 < 1.0.0. Precedence for two pre-release versions with the
+    same major, minor, and patch version MUST be determined by following the
+    precedence rules for pre-release versioning. PEP 440 describes a formal way
+    of matching version numbers, which this specification is compatible with.
+
+## SunPy API Definition
+
+
+The SunPy public API will be defined by the documented behaviour (the content of
+the docstrings) of **any object that is present in the `__all__` list in the
+Python source, and any attributes on those objects not prefixed with one or more
+underscores**, and are therefore included in the API documentation.
+
+A change to the public API is anything that affects not just the name, arguments
+or properties of any object, but any change that alters the behaviour of that
+object (i.e. the values returned from calling it). Deprecating some behaviour is
+a backwards compatible change, and removing it is not backwards compatible.
+Where possible behaviour should be deprecated before it is removed.
+
+# Notes
+
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”,
+“SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this
+document are to be interpreted as described in 
+[RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+# Decision Rationale
+
+
+# Attribution
+
+This document is derived from 
+[Semantic Versioning 2.0.0](http://semver.org/spec/v2.0.0.html) which is
+licensed under the terms of the 
+[Creative Commons Attribution 3.0](https://creativecommons.org/licenses/by/3.0/)
+licence. PEP 440 is in the public domain.
```

Comment:
I'd prefer if the phrase "in the public domain" were not used since it's not a well-defined legal phrase.  I realize that this is just parroting the language in the PEP – see [here](https://github.com/python/peps/issues/123) for their lack of interest in being more precise – so maybe there's a way to make clear that the phrase is their "fault", not ours.